.gitignore

```diff
@@ -1,4 +1,5 @@
 .idea/
+icgc_download_client.egg-info
```

Comment:
Okay then lets use *.egg-info. I'm still confused by the `icgc_download_client` name. That should have been rid from this source tree.